.github/workflows/forms-flow-rsbc-service.yml

```diff
@@ -0,0 +1,52 @@
+name: forms-flow-service-CD
```

Comment:
replace "forms-flow-service" with "forms-flow-rsbcservice" throughout the entire file